packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -574,6 +576,66 @@ void main() {
     }, variant: TargetPlatformVariant.all());
   }
 
+  for (final Curve? snapAnimationCurve in <Curve?>[Curves.easeOut, Curves.bounceOut]) {
+    testWidgets('Widget snaps to nearest snap target with '
+        'snapAnimationCurve: $snapAnimationCurve', (WidgetTester tester) async {
```

Comment:
Here I piggybacked on the tests above for `snapAnimationDuration`. Do you think it's enough or do you need me to add more specific tests that verifies that the position of the sheet follows the curve during a snap animation?

Reply:
It would be good to have a test that doesn't pass on master (minus the new parameter). I think we just need to verify that the DraggableScrollableSheet is actually using the curve that you give it and not acting linearly. Could you check the y position of the widget halfway through the snap animation and expect it to be less than when the linear animation is used? Something like that.

Reply:
Thanks @justinmc! I added a test that verifies that the position follows the given curve at fixed intervals.